command/metrics_test.go

```diff
@@ -0,0 +1,79 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/mitchellh/cli"
+	"github.com/stretchr/testify/require"
+)
+
+var _ cli.Command = &OperatorMetricsCommand{}
+
+func TestCommand_Metrics_Cases(t *testing.T) {
+	t.Parallel()
+
+	srv, _, url := testServer(t, false, nil)
+	defer srv.Shutdown()
+
+	ui := new(cli.MockUi)
```

Comment:
As per https://github.com/hashicorp/nomad/pull/9027: 
```suggestion
	ui := cli.NewMockUi()
```

Reply:
I was reading your comments last night and agree -- fixed!